api/observationunits.py

```diff
@@ -0,0 +1,9 @@
+import json
+
+data = json.load(open('data/observationunits.json', 'r'))
```

Comment:
These json files are just dummy data put in place so the search interface team could know what keys to expect in responses for endpoints. I'm going to rename the folder this is in to make it clear that this folder is not real data. 

As far as whether we should have any data cached or stored, that might be something worth considering if it would speed up queries.

Reply:
that makes sense. Maybe all that would be required is to automate updating these files w/ a curl statement to the api?

Reply:
I think that might be a good idea for the future if caching information would be helpful.